test/unit/core.js

```diff
@@ -464,8 +464,11 @@ QUnit.test( "isNumeric", function( assert ) {
 	assert.ok( t( -16 ), "Negative integer number" );
 	assert.ok( t( 0 ), "Zero integer number" );
 	assert.ok( t( 32 ), "Positive integer number" );
+	assert.ok( t( "0b111110" ), "Binary integer literal" );
```

Comment:
Hey whatever works. I think you'd need something like `+"0o1" === 1` for non-ES6 browsers though?